FlipperZero/Core/View/HomeView.swift

```diff
@@ -36,6 +36,12 @@ struct HomeView: View {
 
 struct HomeView_Previews: PreviewProvider {
     static var previews: some View {
-        HomeView()
+        Group {
+            HomeView()
+            HomeView().body.font(.custom(.boldPixel, size: 50))
+            HomeView().body.font(.custom(.regularPixel))
+            HomeView().body.font(.regularPixel())
+            HomeView().body.font(.regularRoboto())
```

Comment:
This is good to see what happens in this PR, but for later changes it might be confusing.
Could you please create a separate view (something like "palette") that won't be used it the app, but will allow to demo all fonts and in the future other UI?